tests/conftest.py

```diff
@@ -67,6 +66,22 @@ def qubit_device_2_wires():
 def qubit_device_3_wires():
     return qml.device('default.qubit', wires=3)
 
+@pytest.fixture(scope="session")
+def tensornet_device(n_subsystems):
+    return qml.device('expt.tensornet', wires=n_subsystems)
+
+@pytest.fixture(scope="function")
+def tensornet_device_1_wire():
+    return qml.device('expt.tensornet', wires=1)
+
+@pytest.fixture(scope="function")
+def tensornet_device_2_wires():
+    return qml.device('expt.tensornet', wires=2)
+
+@pytest.fixture(scope="function")
+def tensornet_device_3_wires():
+    return qml.device('expt.tensornet', wires=3)
```

Comment:
Are the bottom three redundant? Since you could just use the `tensornet_device` fixture with `@pytest.mark.parametrize("n_subsystems", [1])`, etc.

Reply:
Probably. I just copied how it was implemented in `test_default_qubit.py`